gulpfile.babel.js

```diff
@@ -290,11 +290,11 @@ gulp.task("watch", (cb) => {
 
       // WEBPACK HMR did the job with browser sync
       $.watch(`${paths.site.src}/js/**/*`, () => {
-        runSequence("webpack:lib", "webpack:site");
+        runSequence("webpack:site", "webpack:lib");
```

Comment:
Actually now that i'm thinking about it, we don't need to rebuild lib on site change, it has it has nothing to do with it... not sure why I made this :man_facepalming:

Let's remove `runSequence` here